_extensions/software-licensing/updater-implementation-instantiate.html

```diff
@@ -0,0 +1,70 @@
+---
+layout: sample
+title: Updater Implementation for WordPress Plugins -- Instantiating the Updater Class
+description: Sample code to illustrate how to instantiate the Software Licensing plugin updater class in your plugin.
+collection: extensions
+category: software-licensing
+---
+
+<?php
+
+/*
+Plugin Name: Sample Plugin
+Plugin URI: http://pippinsplugins.com/
+Description: Illustrates how to include an updater in your plugin for EDD Software Licensing
+Author: Pippin Williamson
+Author URI: http://pippinsplugins.com
```

Comment:
Should we change these URL/author references to `https://sandhillsdev.com` and `Sandhills Development, LLC` ?